x-pack/plugins/xpack_main/server/lib/telemetry/local/get_local_stats.js

```diff
@@ -18,16 +19,21 @@ import { getXPack } from './get_xpack';
  * @param {Object} xpack License and X-Pack details
  * @return {Object} A combined object containing the different responses.
  */
-export function handleLocalStats(clusterInfo, clusterStats, xpack) {
-  return {
+export function handleLocalStats(server, clusterInfo, clusterStats, license, xpack, kibana) {
+  const stats = {
     timestamp: (new Date()).toISOString(),
     cluster_uuid: get(clusterInfo, 'cluster_uuid'),
     cluster_name: get(clusterInfo, 'cluster_name'),
     version: get(clusterInfo, 'version.number'),
     cluster_stats: omit(clusterStats, '_nodes', 'cluster_name'),
     collection: 'local',
-    ...xpack
```

Comment:
Any reason to not just keep the object return and handle it all in one spot rather than setting, then mutating it?

```diff
-    ...xpack
+    license,
+    stack_stats: {
+      kibana: handleKibanaStats(server, kibana),
+      xpack,
+    },
```

Reply:
No reason :) I had an earlier incarnation of the code where each possibly undefined value had a check before it would be added, so that the result would not have any keys with `undefined`. After changing the unit tests quite a bit, keeping it clean wasn't as important.